Mage/src/main/java/mage/abilities/effects/common/ExileSourceEffect.java

```diff
@@ -65,7 +65,7 @@ public boolean apply(Game game, Ability source) {
                 Card sourceCard = (Card) sourceObject;
                 return controller.moveCardsToExile(sourceCard, source, game, true, exileZoneId, exileZoneName);
             }
-            return true;
+            return false;
```

Comment:
these changes make sense to me. it is theoretically possible it breaks some other card, but that's unlikely and the fault of the other card, so I say it's fine as no test failures.